pom.xml

```diff
@@ -440,7 +440,7 @@
     <swagger-parser-version>1.0.6-SNAPSHOT</swagger-parser-version>
     <scala-version>2.10.4</scala-version>
     <felix-version>2.3.4</felix-version>
-    <swagger-core-version>1.5.1-M2</swagger-core-version>
+    <swagger-core-version>1.5.2-M2-SNAPSHOT</swagger-core-version>
```

Comment:
That is why I've referenced the swagger-api/swagger-core/pull/1002 pull request. The  com.wordnik.swagger.models.Response from that change is required. Just don't know how to reference the develop_2.0 branch properly.